ds-collector/ds-collector

```diff
@@ -571,14 +580,14 @@ collect_storage() {
   #for DEVICE in $(cat "$configHome/cassandra.yaml" | sed -n "/^data_file_directories:/,/^$/p" | grep -E "^.*-" | awk '{print $2}')
   for device in $(echo "$cassandra_data_dir" | awk '{gsub(/,/,"\n");print}')
   do
-      dm="$(df -h "$device" | grep -v "Filesystem" | awk '{print $1}')"
+      dm="$(df $DF_OPT -h "$device" | grep -v "Filesystem" | awk '{print $1}')"
```

Comment:
Yeah, like here... except that `DF_OPT` is only being set in the function contexts above (and not `export`ed), so this is going to evaluate to `""`

Reply:
yes, i was thinking it would carry from node_checks() to here, but you are right, it won't. fixing…

Reply:
fixed